plugins/sslservices/sslservices.cpp

```diff
@@ -120,29 +120,29 @@ class OpenSSLCache
         if (newObj)
         {
             cache.emplace_front(algorithm_name, newObj);
-            if (cache.size() > OPENSSL_MAX_CACHE_SIZE)
+            if (cache.size() > SSLSERVICES_MAX_CACHE_SIZE)
                 cache.pop_back();
         }
         else
             failOpenSSLError("adding new object to cache");
 
         return newObj;
-    };
+    }
 
-    void printStatistics() {DBGLOG("OPENSSL %s CACHE STATS: HITS = %d, MISSES = %d", cacheName.c_str(), hits, misses);};
+    void printStatistics() {DBGLOG("SSLSERVICES %s CACHE STATS: HITS = %d, MISSES = %d", cacheName.c_str(), hits, misses);}
 
     void init()
     {
         setCacheName();
         hits = 0;
         misses = 0;
-    };
+    }
 
-    void clear() {cache.clear();};
+    void clear() {cache.clear();}
 
 private:
-    int hits;
-    int misses;
+    size32_t hits;
```

Comment:
this should be unsigned.  size32_t is only used for numbers of bytes - not arbitrary counts.